### PR TITLE
Concepto Moneda frontend implementation

### PR DESCRIPTION
**Pull request by neto3000**

### CODE REVIEW

File: app/models/sat/catalogo/base.rb [hunk 1]
```diff
@@ -4,10 +4,11 @@ class Base < ActiveRecord::Base
       self.abstract_class = true
 
       def self.search(keyword)
+        keyword.empty? ? limite = nil : limite = 20
         conditions = searchable_columns.map do |column|
           "#{column} like :keyword"
         end.join(' OR ')
-        where(conditions, keyword: "%#{keyword}%").limit(20)
+        where(conditions, keyword: "%#{keyword}%").limit(limite)
```
Comment (igmarin) [inline app/models/sat/catalogo/base.rb]: keep english not spanish in this project

Comment (igmarin) [inline app/models/sat/catalogo/base.rb]: Why not only do:  `where(conditions, keyword: "%#{keyword}%")&.limit(20)` & remove line 7

**Review by jtapia — APPROVED** [review]
LGTM

**Review by igmarin — APPROVED** [review]
LGTM

**Pull request merged**